app/views/conference/_venue.html.haml

```diff
@@ -6,12 +6,21 @@
 -else
   .container
     .row
-      .col-md-12
+      - if @conference.venue.commercial.nil?
+        .col-md-6
+          %img{ "data-src" => "holder.js/500x300?text=No Commercial Set", class: 'img-responsive img-rounded' }
```

Comment:
Please do not show placeholder images to visitors. If there is no commercial, just show the address